tests/integration/solidity_contracts/PlainOpcodes/test_plain_opcodes.py

```diff
@@ -35,6 +36,39 @@ async def test_should_return_self_address(
                 == int(evm_contract_address, 16)
             )
 
+    class TestBlockhash:
+        async def test_should_return_blockhash(
+            self,
+            deploy_solidity_contract: Callable,
+            addresses,
+            blockhashes,
+        ):
+            counter = await deploy_solidity_contract(
+                "Counter", caller_address=addresses[1]["int"]
+            )
+            counter_address = Web3.toChecksumAddress(
+                "0x"
+                + hex(
+                    counter.contract_account.deploy_call_info.result.evm_contract_address
+                )[2:].rjust(40, "0")
+            )
+            integration_contract = await deploy_solidity_contract(
+                "PlainOpcodes",
+                counter_address,
+                caller_address=addresses[1]["int"],
+            )
+
+            blockhash = await integration_contract.opcodeBlockHash(503594)
```

Comment:
Will this fail in a few days?

Reply:
So if we update the `blockhashes.json` file by running `make get-blockhashes`, this test will have to be updated because in the `starknet` fixture, we update the block number using this file like so 
<img width="745" alt="image" src="https://user-images.githubusercontent.com/35031356/207431994-91ec3f14-a9a0-4aa9-b6c9-3e721cd9aa86.png">

Otherwise, the test will succeed as the block number will be `503596`, as per the current block number from the file

Reply:
Indeed that's true! I've updated this in the new pr so we use `mock_blockhashes.json` instead of hardcoded values 👍  Thanks Elias!